avocado_vt/plugins/vt_joblock.py

```diff
@@ -68,6 +68,24 @@ def _create_self_lock_file(self, job):
         except Exception as e:
             raise LockCreationError(e)
 
+    def _get_lock_files(self):
+        """
+        Get the list of lock file names under the current lock dir
+
+        :returns: a list with the full path of files that match the
+                  lockfile pattern
+        :rtype: list
+        """
+        try:
+            files = os.listdir(self.lock_dir)
+            pattern = re.compile(r'avocado-vt-joblock-[0-9a-f]{40}-[0-9]+'
+                                 '-[0-9a-z]{8}\.pid')
+            return [os.path.join(self.lock_dir, _) for _ in files
```

Comment:
Well it's mentioned in the documentation, glob is __really simple__ implementation, but for me `glob.glob(os.path.join(self.lockdir, "avocado-vt-joblock-*"))` would suffice. Anyway let's live with this version for now...